Update udfs.md (#8317)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/udfs.md

## Defining UDFs in dbt

You can define SQL and Python UDFs in dbt. Python UDFs are currently supported in Snowflake and BigQuery when using <Constant name="core" />. 

> :::tip

Reviewer: ```suggestion
:::important UDF support
```

> Keep the following considerations in mind when developing UDFs:

Reviewer: ```suggestion
When developing UDFs, it's important to understand the following support limitations:
```

- Python UDFs aren't yet supported in <Constant name="fusion" />.
- Additional languages (for example, Java, JavaScript, Scala) aren't currently supported.

> See the [Limitations](#limitations) section below for the full list of currently supported UDF capabilities.

Reviewer: This section is more about limitations than what is supported. It might be helpful to readers to put a lot of this information in the prerequisites up top than in a limitations section at the bottom

"You must use SQL UDFs on Fusion (Python UDF support coming soon)" for example
```suggestion
See the [Limitations](#limitations) section below for the full list of currently supported UDF capabilities.
```

Author: good shout! i'm not the original author of this page but how about i meet you halfway andmove the callout above, after prerequisites? python udfs aren't necessarily a requirement to develop udfs - it's just if they are developing udfs with python, it's not supported in fusion yet.

## Defining UDFs in dbt

You can define SQL and Python UDFs in dbt. Python UDFs are currently supported in Snowflake and BigQuery when using <Constant name="core" />. 

:::tip
Keep the following considerations in mind when developing UDFs:

> - Python UDFs are supported only on Snowflake and BigQuery when using <Constant name="core" />.

Author: ```suggestion
```
